src/platform/C4WindowGTK.cpp

```diff
@@ -378,6 +378,34 @@ static gboolean fullscreen_restore(gpointer data)
 	return FALSE;
 }
 
+static bool mouse_was_grabbed = false;
```

Comment:
Can this be turned into a member variable of C4Window instead of a global variable? The `user_data` parameter in OnFocusInFS is a pointer to the C4Window instance.